dependencies/pom.xml

```diff
@@ -28,7 +28,6 @@
     <module>javamail</module>
     <module>jaxb</module>
     <module>jcifs</module>
-    <module>jersey</module>
     <module>jersey-client</module>
```

Comment:
features/reporting/repository still relies on it too. We could work on migrating these to use cxf though.